Formula/chapel.rb

```diff
@@ -14,36 +13,46 @@ class Chapel < Formula
     sha256 x86_64_linux: "7cf01213c80d41ed1b20451fd9f11b4b651a1b9a603729826a383e73475918b3"
   end
 
+  depends_on "gmp"
   depends_on "llvm@11"
   depends_on "python@3.9"
 
+  patch :DATA
```

Comment:
Just one more thing:
```suggestion
  # can be removed after https://github.com/chapel-lang/chapel/pull/18880 merged
  patch :DATA
```